### ISSUE TITLE
Pause/Break not working in Attach mode in VSC

### ISSUE THREAD

**Issue by DonJayamanne** [Microsoft/ptvsd#460]
**Steps to reproduce**
* Launch remote program (`python one.py`)
* Attach
* Next hit the `Pause/Break` button
* Nothing happens

```python
import time
import ptvsd
print(ptvsd.__version__)
print(ptvsd.__file__)
ptvsd.enable_attach()
for i in range(100):
    time.sleep(1)
    print(i)
```

**Note: This approach works**
* Launch remove program (`python -m ptvsd --server --port one.py`)
* Attach
* Next hit the `Pause/Break` button
* Debugger breaks inside the loop

```python
import time
for i in range(100):
    time.sleep(1)
    print(i)
```

**Comment by karthiknadig** [comment]
The second approach did not work for me.

**Comment by DonJayamanne** [comment]
@karthiknadig 
oops, the command should be `python -m ptvsd --server --port 5678 one.py`

**Comment by karthiknadig** [comment]
I tested with port number. When i said it did not work, I meant i ran into the same issue as the first case. Pause/Break does not work after attaching.

**Comment by DonJayamanne** [comment]
> Pause/Break does not work after attaching.

This is a regression. It used to work on all platforms. I tested this on all platforms. Looks like something is wrong on Windows. Are you testing with vscode?

**Comment by karthiknadig** [comment]
It repros with VSC on windows.

**Comment by ericsnowcurrently** [comment]
@DonJayamanne, this appears to be fixed with #527.  Could you verify?

**Comment by ericsnowcurrently** [comment]
I've verified that it's fixed.  If you notice any issues, feel free to re-open.
